cstrml/staking/src/tests.rs

```diff
@@ -2774,11 +2774,11 @@ fn garbage_collection_on_window_pruning() {
 }
 
 #[test]
-// TODO: this slashing test case should be changed
-/*fn slashing_guarantors_by_span_max() {
+fn slashing_guarantors_by_span_max() {
     ExtBuilder::default().build().execute_with(|| {
-        start_era(1);
-        start_era(2);
+        start_era(1, false);
+        start_era(2, false);
+        start_era(3, false);
```

Comment:
tests are changed in this PR. https://github.com/crustio/crust/pull/89/files#r439804070

Reply:
Seems reasonable for me